mindsdb/interfaces/agents/mindsdb_database_agent.py

```diff
@@ -4,36 +4,38 @@
 """
 from typing import Any, Iterable, List, Optional
 
-from mindsdb.interfaces.skills.skill_tool import skill_tool
 from mindsdb.utilities import log
 from langchain_community.utilities import SQLDatabase
+from mindsdb.interfaces.skills.sql_agent import SQLAgent
 
 logger = log.getLogger(__name__)
 
 
 class MindsDBSQL(SQLDatabase):
+    @staticmethod
+    def custom_init(
```

Comment:
what is the purpose of using this function? what is difference with previous behavior?